### DISCUSSION TITLE
[4주차] 온라인 스터디 (면접)

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#32] [General]
### 진행순서
~ 20:15 카훗`Kahoot`으로 입풀기
~ 21:00 1:N 모의면접 진행
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 😂
* 좋았던 점
* 건의사항
* 느낀점

**Reply by ChaerinYu** [reply]
* 느낀점
스터디장으로서 많이 부족하네요..^-^ 다음 면접 때는 정말 꼭.. 열심히 준비해보도록 할게요... ㅠㅠ

**Reply by KJY97** [reply]
- 느낀점
  면접 말할 때 너무 긴장하고 잘 말 못한거 같습니다..ㅋㅋㅋ 열심히 할게요!

**Reply by doho-ho** [reply]
- 느낀점
   재밌었는데, 면접 끝나면 질문 내용 정리해서 올리는건 어떨까요?

**Reply by ChaerinYu** [reply]
> * 끝나면 질문 내용 정리해서 올리는건 어떨까요?

다음부터 질문 내용들은 `discussion`에 작성해보도록 하겠습니다!
